mvp-spec/partials-updater/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "start": "tsc && node dist/index.js"
+    "start": "ts-node src/index.ts"
```

Comment:
What is the advantage of doing so ?

Reply:
- I believe it builds only what is necessary to run this particular `index.js`, ignoring potential other files that are not needed
- generated on the fly, so it doesn't generate a `dist` folder ;-)